Airport_Ticket_Booking_System_Test/FlightServiceTest.cs

```diff
@@ -0,0 +1,49 @@
+// File: Airport_Ticket_Booking_System_Test/FlightServiceTests.cs
+using System;
+using System.Collections.Generic;
+using Xunit;
+using Airport_Ticket_Booking_System.Models;
+using Airport_Ticket_Booking_System.Interfaces;
+using Airport_Ticket_Booking_System.Services;
+
+namespace Airport_Ticket_Booking_System_Test
+{
+    public class FlightServiceTests
+    {
+        private readonly IFlightService _flightService;
+
+        public FlightServiceTests()
+        {
+            _flightService = new FlightService();
+
+            var flight1 = new Flight("FL001", "USA", "Canada", "JFK", "YYZ", DateTime.UtcNow, new Dictionary<string, decimal> { { "Economy", 500M } });
+            var flight2 = new Flight("FL002", "UK", "France", "LHR", "CDG", DateTime.UtcNow.AddDays(1), new Dictionary<string, decimal> { { "Economy", 400M } });
+
+            _flightService.AddFlight(flight1); 
+            _flightService.AddFlight(flight2);
+        }
+
+        [Fact]
+        public void GetAllFlights_ShouldReturnAllFlights()
```

Comment:
you can add third part WhenFlightsExist

Reply:
you can add another test case when flight list is empty
in this case you may extract the constructor logic in a method and call it when needed